sdk/digitaltwins/Azure.DigitalTwins.Core/tests/QueryBuilderTests/FullQueryTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.DigitalTwins.Core.QueryBuilder;
+using Azure.DigitalTwins.Core.Queries.QueryBuilders;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Azure.DigitalTwins.Core.Tests.QueryBuilderTests
+{
+    public class FullQueryTests
+    {
+        [Test]
+        public void SelectAllTests()
```

Comment:
@azabbasi we should teach Nick about how unit tests get hard to identify when you get a lot of them, and the naming scheme we've come up with elsewhere to improve upon that.

IOW in this case it'd be `AdtQueryBuidler_Select_AllSimple` or something like that.